examples/Generator/inverse_trainer_xyz.py

```diff
@@ -0,0 +1,102 @@
+import os
+from pathlib import Path
+
```

Comment:
Maybe some description what this file should showcase? When it is example it would be nice it it guides the user through what is going on. The name othe the file is not self-explanatory.

* What is the differentce in purpose of `inverse_trainer.py` and `generator_trainer.py` ?

Reply:
I intend to add proper tutorial pages for the generators, similar to the recent AFMulator tutorial, so I will leave this to later. But briefly:
- `inverse_trainer`: The original generator that only works with xyz files on the disk. Simple but limited.
- `generator_trainer`: The updated generator that works with any kind of input, but requires manual programming for loading the files from the disk and feeding them to the generator.